dev_docs/tutorials/kibana_page_template.mdx

```diff
@@ -84,3 +84,36 @@ When passing both a `pageHeader` configuration and `isEmptyState`, the component
 ```
 
 ![Screenshot of demo custom empty state code with a page header. Shows the Kibana navigation bars, a level 1 heading "Dashboards", and a centered empty state with the a level 2 heading "No data", body text "You have no data. Would you like some of ours?", and a button that says "Get sample data".](../assets/kibana_header_and_empty_state.png)
+
+## `solutionNav`
+
+To add left side navigation for your solution, we recommend passing [**EuiSideNav**](https://elastic.github.io/eui/#/navigation/side-nav) props to the `solutionNav` prop. The template component will then handle the mobile views and add the solution nav embellishments. On top of the EUI props, you'll need to pass your solution `name` and an optional `icon`.
+
+If you need to custom side bar content, you will need to pass you own navigation component to `pageSideBar`. We still recommend using [**EuiSideNav**](https://elastic.github.io/eui/#/navigation/side-nav).
+
+When using `EuiSideNav`, root level items should not be linked but provide section labelling only.
+
+```tsx
+<KibanaPageLayout
+  solutionNav={{
+    name: 'Management',
+    icon: 'managementApp',
+    items: [
+      {
+        name: 'Root item',
+        items: [
+          { name: 'Navigation item', href: '#' },
+          { name: 'Navigation item', href: '#' },
+        ]
+      }
+    ]
+  }}
+>
+  {...}
+</KibanaPageLayout>
+```
+
+
+![Screenshot of Stack Management empty state with a provided solution navigation.](../assets/kibana_template_solution_nav.png)
```

Comment:
Adding some more notes describing the image that might be useful to low-vision users to piece together the image. 

```suggestion
![Screenshot of Stack Management empty state with a provided solution navigation shown on the left, outlined in pink.](../assets/kibana_template_solution_nav.png)
```